### DISCUSSION TITLE
I am using flask cli. How to callback after run() function?

### DISCUSSION THREAD

**Discussion by youyinnn** [pallets/flask#5644] [Q&A]
Literaturly, How???
flask cli take over the \_\_main\_\_. How am I supposed to run something that needs the __main__ guard?

**Comment by RonnyPfannschmidt** [comment]
Could you elaborate on what the intent of that code is

Depending on that, theres multiple approaches

**Reply by youyinnn** [reply]
Nvm. I decided to not using flask in that way.
